lib/examples/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _examples_index:
+
+Debops examples index
+=====================
+
+Aim of examples
+---------------
+
+DebOps examples are DebOps projects that contain only inventory files.
+They are provided to DebOps users to:
+
+- describe a specific configuration (scenario): recipes
+- use as integration tests
+
+
+Design of examples
+------------------
+
+Examples are written and tested in Vagrant environment used in
+DebOps. Each example directory contain a Vagrantfile that can be use to start
+VMs in this directory.
+
+To simplify provisioning and to leverage auto-generated Ansible inventory in
+Vagrant environement, examples use only groups in inventory (instead of
+hosts).
+
+
+Provisioning examples in Vagrant
+--------------------------------
+
+After doing a :cmd:`vagrant up` in an example directory, you can provision
+examples with following command:
+
+..  code-block:: console
+
+    vagrant provision --provision-with=setup_examples
```

Comment:
Perhaps you could change the name of the provision script to `examples`, then the command would be:
```sh
vagrant provision --provision-with=examples
```
Seems a bit nicer to use and easier to remember.

Reply:
Agree, I try to follow name of current provisionners (like `setup_eatmydata`).

Reply:
I got your point, you mean only change name of provisionner not name of script.